GVFS/GVFS.Installer.Mac/CreateMacInstaller.sh

```diff
@@ -103,13 +103,54 @@ function CreateInstaller()
     eval $pkgBuildCommand || exit 1
 }
 
+function CreateMetaInstaller()
+{
+    GITVERSION="$($VFS_SCRIPTDIR/GetGitVersionNumber.sh)"
+    GITDMGPATH="$(find $VFS_PACKAGESDIR/gitformac.gvfs.installer/$GITVERSION -type f -name *.dmg)" || exit 1
+    GITDMGNAME="${GITDMGPATH##*/}"
+    GITINSTALLERNAME="${GITDMGNAME%.dmg}"
+    GITVERSIONSTRING=`echo $GITINSTALLERNAME | cut -d"-" -f2`
+    
+    if [[ -z "$GITVERSION" || -z "$GITVERSIONSTRING" ]]; then
+        echo "Error creating metapackage: could not determine Git package version."
+        exit 1
+    fi
+    
+    if [ ! -f "$GITDMGPATH" ]; then
+        echo "Error creating metapackage: could not find Git disk image."
+        exit 1
+    fi
+    
+    mountCmd="/usr/bin/hdiutil attach \"$GITDMGPATH\""
+    echo "$mountCmd"
+    eval $mountCmd || exit 1
+    
+    MOUNTEDVOLUME=`/usr/bin/find /Volumes -type d -name "Git $GITVERSIONSTRING*"`
+    GITINSTALLERPATH=`/usr/bin/find "$MOUNTEDVOLUME" -type f -name "git-$GITVERSIONSTRING*.pkg"`
+    METAPACKAGENAME="$INSTALLERPACKAGENAME-Git.$GITVERSION.pkg"
+    
+    if [ ! -f "$GITINSTALLERPATH" ]; then
+        echo "Error creating metapackage: could not find Git installer package."
+        exit 1
+    fi
+    
+    buildMetapkgCmd="/usr/bin/productbuild --package \"$GITINSTALLERPATH\" --package \"${BUILDOUTPUTDIR}\"$INSTALLERPACKAGENAME.pkg \"${BUILDOUTPUTDIR}\"$METAPACKAGENAME"
```

Comment:
What is the user experience like with this .pkg?  Does it walk them through the Git and VFS4G installers, or does it just package both together (and the user would run them individually)?

Reply:
The user wouldn't know there are two separate installers. It will appear like a single installer that installs both VFSForGit and Git.